.github/workflows/behat-stability-2.yml

```diff
@@ -32,6 +32,13 @@ jobs:
           coverage: none # disable xdebug, pcov
           tools: cs2pr
 
+      # We want to reuse our project setup to run different jobs at once.
+      - uses: actions/cache@v3
```

Comment:
@tbsiqueira We don't need this in here. Please read more about cache here: https://github.com/actions/cache. This addition has no effect on tests or its setup.